catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorAddVariableTest.java

```diff
@@ -88,4 +88,19 @@ public void addListTest() {
 		onDataList()
 				.onListAtPosition(0).checkHasName(listName);
 	}
+
+	@Category({Cat.AppUi.class, Level.Smoke.class})
+	@Test
+	public void cancelOnTouchCancelButtonTest() {
```

Comment:
Please rename this function to something like "cancelAddVariableTest", so future developers know what this function does by it's name